### ISSUE TITLE
Method does not override or implement a method from a supertype

### ISSUE THREAD

**Issue by dangbh1002** [react-community/react-native-image-picker#666]
### Description
Error in file ImagePickerPackage.java
```
\android\src\main\java\com\imagepicker\ImagePickerPackage.java:34 error: method does not override or implement a method from a supertype
@Override
```

### Solution

1. Comment line 34 "@Override" before function createJSModules()
2. cd android -> gradlew clean

**Comment by m30do** [comment]
+1

**Comment by radko93** [comment]
This has been already fixed.

**Comment by janicduplessis** [comment]
Fixed in 0.26.4
